packages/loopring_v3/contracts/amm/libamm/AmmExitProcess.sol

```diff
@@ -111,6 +111,11 @@ library AmmExitProcess
                     "INVALID_TX_DATA"
                 );
 
+                // Replay protection when using a signature (otherwise the approved hash is cleared onchain)
+                if (signature.length > 0) {
+                    require(transfer.storageID == exit.storageIDs[i], "INVALID_TX_DATA");
+                }
```

Comment:
Seems like this was removed even when a signature is used. Unless some other onchain replay protection is implemented (which will be more expensive) there is no other way here to prevent replay protection (even EdDSA signed txs, which could in theory be used for the join transfers, are impossible because the pool account has no EdDSA public key).